### ISSUE TITLE
IgxSingleCombo: strange tab behavior when 2 single combos are next to each other

### ISSUE THREAD

**Issue by Timmeeeey** [IgniteUI/igniteui-angular#14791]
## Description  
If you have 2 single combos next to each other that have a selected value and then click on the first one and press tab, then instead of just closing the dropdown the focus jumps to the next single combo. Each following tab doesn't do anything. The focus stays on the second combo.

 * igniteui-angular version: 18.1.11

## Steps to reproduce  

Normal behavior:
Pressing tab only closes the dropdown

![2024-09-18_15h21_34](https://github.com/user-attachments/assets/bd5e23fa-a8e4-4aa1-9b95-34a7acd47e5e)

Wrong behavior:
Pressing tab focuses second dropdown. When pressing tab again, nothing happens.

![2024-09-18_15h22_15](https://github.com/user-attachments/assets/21663f32-0483-4c54-88c3-10b8845013b8)

## Attachments  
https://stackblitz.com/edit/wpppeb

**Comment by Hristo313** [comment]
@Timmeeeey What is described as "wrong behavior" is actually the expected behavior according to this comment: https://github.com/IgniteUI/igniteui-angular/issues/14085#issuecomment-2310432644. The normal behavior should ensure that pressing the Tab key moves focus to the next dropdown.

**Comment by Timmeeeey** [comment]
@Hristo313 It's ok that the focus moves to the next dropdown.
The problem is that when pressing tab again, nothing happens. The focus is trapped in the second dropdown.

**Comment by Hristo313** [comment]
@Timmeeeey Yes, and if the dropdown is opened, pressing Tab should not only close it but also focus on the next dropdown.
